zcash_client_backend/CHANGELOG.md

```diff
@@ -81,6 +86,15 @@ and this library adheres to Rust's notion of
   - `PROPOSAL_SER_V1`
   - `ProposalDecodingError`
   - `proposal` module, for parsing and serializing transaction proposals.
+  - `impl TryFrom<&CompactSaplingOutput> for CompactOutputDescription`
+  - `impl TryFrom<&CompactOrchardAction> for CompactAction`
+  - `CompactOrchardAction::{cmx, nf, ephemeral_key}`
+- `zcash_client_backend::scanning`:
+  - `ScanningKeyOps` has replaced the `ScanningKey` trait.
+  - `ScanningKey` is now a concrete type that bundles an incoming viewing key
+    with an optional nullifier key and key source metadata.
```

Comment:
This is confusing to be in the "Added" section without a corresponding entry in "Removed", but that also is confusing because users' existing named imports won't cause "cannot find X" compiler errors, and the error they get instead will direct them to look for changes. Move this entry to "Changed", and reference `ScanningKeyOps` therein.

Reply:
:heavy_check_mark: